src/core/data-structures/globals.d.ts

```diff
@@ -0,0 +1,7 @@
+import './amp-globals.d';
```

Comment:
FMI: the idea is that every globals file will import amp-globals?

Reply:
Every one that has it, yes. This lets globs specify `**/globals.d.ts` while relying on `globals.d.ts` to pull in any amp-specific globals. If/when core is extracted into Bento or another repo, the ties with `amp-globals` can be cut.

Reply:
Seems pretty equivalent to adding an amp-globals glob that we wouldn't port.

Reply:
It would either be here or the TSConfig files. I don't feel strongly about which